src/main/java/seedu/address/model/leave/Leave.java

```diff
@@ -68,6 +68,32 @@ public Leave(Person employee, String title, Date start, Date end)
         this.status = Status.getDefault();
     }
 
+    /**
+     * Constructor for Leave object with status.
+     *
+     * @param employee
+     * @param title
+     * @param start
+     * @param end
+     * @param description
+     * @param status
+     */
+    public Leave(Person employee, String title, Date start, Date end, String description, Status status)
+            throws EndBeforeStartException {
+
+        requireAllNonNull(employee, title, description, start, end, status);
+
+        if (end.isBefore(start)) {
+            throw new EndBeforeStartException();
```

Comment:
Strange that this line is flagged out for not being covered - I thought there was already a test that covers this branch